diff --git a/node_modules/@agoric/ui-components/dist/components/NatPurseSelector.js b/node_modules/@agoric/ui-components/dist/components/NatPurseSelector.js
index 2f14007..c3d9f22 100644
--- a/node_modules/@agoric/ui-components/dist/components/NatPurseSelector.js
+++ b/node_modules/@agoric/ui-components/dist/components/NatPurseSelector.js
@@ -1,4 +1,4 @@
-// @ts-check
+// @ts-nocheck
 import { AssetKind } from '@agoric/ertp';
 import clsx from 'clsx';
 import { stringifyValue } from '../display/index.js';
